spec/player_turn_spec.rb

```diff
@@ -1,20 +1,20 @@
 require "spec_helper"
-require_relative "../lib/player_input.rb"
+require_relative "../lib/player_turn.rb"
 require_relative "../lib/coordinates.rb"
 require_relative "../lib/grid.rb"
 require_relative "../lib/printer.rb"
 
-RSpec.describe PlayerInput do
+RSpec.describe PlayerTurn do
   describe "#get_valid_coordinate" do
      it "repeatedly asks for valid coordinate, returns the value when valid" do
       invalid_format = StringIO.new
       grid = Grid.new
       printer = Printer.new(invalid_format)
-      player_input = PlayerInput.new(invalid_format, grid, printer)
+      player_turn = PlayerTurn.new(invalid_format, grid, printer)
       allow(invalid_format).to receive(:gets).and_return("A6", "B7", "A1")
-      coordinates = Coordinates.new(player_input.get_user_input, double(grid))
+      coordinates = Coordinates.new(player_turn.get_initial_input)
```

Comment:
Why is this line needed? `get_initial_input` should be a private method. But also, the gets mocking should setup the input stream to do what you want.

Reply:
Ah, right. Whoops! Fixed.